library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeStateMultiValue.java

```diff
@@ -71,7 +71,6 @@ public ManagedTimeStateMultiValue(@NotNull ManagedTimeStateImpl store, boolean i
   {
     this();
     this.store = Preconditions.checkNotNull(store);
-    this.store.asyncReadSource = Bucket.ReadSource.READERS;
     this.isKeyContainsMultiValue = isKeyContainsMultiValue;
```

Comment:
@chaithu14 please fix ManagedTimeStateMultiValue. Also once you fix failing tests, please create tests ManagedTimeStateMultiValue
There was no need of creating a variable asyncReadSource in AbstractManagedState.